publish.gradle

```diff
@@ -208,6 +208,15 @@ ext.configurePublish = { jnaForTestConfiguration = null,
         }
     }
 
+    task checkMavenArtifacts
+
+    publishing.publications.withType(MavenPublication).each {publication ->
+        def checkFileSizeTask = task "checkLibSizeForMavenArtifact-${publication.artifactId}"(type: Exec) {
+            commandLine "${rootProject.projectDir}/automation/check_artifact_size.sh", project.buildDir, publication.artifactId
+        }
+        checkMavenArtifacts.dependsOn(checkFileSizeTask)
+    }
+
```

Comment:
I've no idea whether this is the best place to do such a check, I just worked backwards from the tasks executed by `decision_task.py` to find a likely-looking spot...

Reply:
This looks fine to me I guess, maybe someone more comfortable with Gradle could step in to see if that's a good place to place this check.